### PR TITLE
Instead of showing not authenticated, redirect user instead

### PR DESCRIPTION
**Pull request by jasonmorais**

### CODE REVIEW

File: ui-community/src/components/shared/require-auth.tsx [hunk 1]
```diff
@@ -39,21 +43,12 @@ const RequireAuth: React.FC<RequireAuthProps> = (props) => {
...
+    return <Navigate to="/" />;
   } else {
-    return (
-      <Row justify={'center'} style={{ height: '100vh', alignItems: 'center' }}>
-        <Space size={'large'} direction="vertical" style={{ textAlign: 'center' }}>
-          <Typography.Title level={2}>Not Authorized</Typography.Title>
-        </Space>
-      </Row>
-    );
+    redirectUser();
```
[the remark this answers is not in the document]
Comment (jasonmorais) [reply ui-community/src/components/shared/require-auth.tsx:51]: That check is implied by the other two if statements above, it cant reach here anyway

File: ui-community/src/components/layouts/accounts/components/community-list.tsx [hunk 2]
```diff
@@ -29,16 +29,11 @@ export const CommunityList: React.FC<CommunityListProps> = (props) => {
...
     });
-    if (filteredCommunities.length > 0) {
-      setDisplayNotFound(false);
-      setCommunityList(filteredCommunities);
-    } else {
-      setDisplayNotFound(true);
-      setCommunityList(props.data.communities);
-    }
+    setDisplayNotFound(filteredCommunities.length === 0);
+    setCommunityList(filteredCommunities.length > 0 ? filteredCommunities : props.data.communities);
```
[the remark this answers is not in the document]
Comment (jasonmorais) [reply ui-community/src/components/layouts/accounts/components/community-list.tsx]: didnt know about this, fixed. Good thing to pickup

File: ui-community/src/components/layouts/accounts/components/community-list.tsx [hunk 3]
```diff
@@ -29,16 +29,11 @@ export const CommunityList: React.FC<CommunityListProps> = (props) => {
...
   };
 
-  const useCommunnityColumns = (props: CommunityListProps) =>
+  const useCommunnityColumns = () =>
```
[the remark this answers is not in the document]
Comment (jasonmorais) [reply ui-community/src/components/layouts/accounts/components/community-list.tsx]: fixed

**Comment by nguyenduy** [comment]
1. When no communities are found, should not show the list.

**Review by nguyenduy — APPROVED** [review]

**Pull request merged**